tests/machine/test_machine.py

```diff
@@ -7,6 +7,11 @@
 from smsdk.smsdk_entities.machine.machine import Machine
```

Comment:
Update the tests of all the functions you changed to use both methods.

Reply:
I did most of the cases. Added new test cased to cover the new styled APIs.